infra.bs

```diff
@@ -216,8 +217,11 @@ in parentheses. [[!UNICODE]]
 
 <p>In certain contexts <a>code points</a> are prefixed with "0x" instead of "U+".
 
-<p>A <dfn export>scalar value</dfn> is a <a>code point</a> that is not in the range
-U+D800 to U+DFFF, inclusive.
+<p>A <dfn export>surrogate code point</dfn> is a <a>code point</a> that is in the range U+D800 to
```

Comment:
Hmm, so then I don't quite understand why "Per definition these are isolated surrogates." later is true. What is special about the surrogates inside a JavaScript string that makes them always isolated?